release-3.7.0-announcement.md

```diff
@@ -0,0 +1,31 @@
+We're excited to announce the availability of Jamulus 3.7.0!
+
+This release re-organizes Server lists to make room for more servers:
+- Default has been renamed to Any Genre 1.
+- Any Genre has been renamed to Any Genre 2.
+- Any Genre 3 has been added.
+- Choral/Barbershop has been added.
+- Classical/Folk/Choral has been renamed to Classical/Folk.
+
+Besides that, tens of usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Please find all the details in the [Changelog](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0).
+
+Downloads:
+- [Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe)
```

Comment:
Ideally, these links would be SEO-friendly maybe. So jamulus.io/download/windows.html etc which redirect to the relevant GitHub URLs. Not sure if we can do that though, but it means that we can have Google present a "download" sub-link (or whatever they call it) on search results. I'm in two minds about pushing downloads so directly without having people read some form of orientation first, but it's a convention on other sites to have a big download call to action.

Reply:
Happy to change that, but what you propose doesn't exist yet, right? So the question would be: Is it impotant enough to add such pages and delay the release until then or is this rather something for the next release?

I think a direct link to an .exe file is way better than the previous SourceForge download landing pages. If we had a concise download page with content by us, this would be even better for sure. :)

Reply:
Yes, doesn't exist yet. No point in delaying the release for it. We are working on a dedicated "download" page for the site, so let's wait for that.

Reply:
> BTW I guess this means (for now) that we will point at GH for the direct links?

You mean the "Download now" link on the start page? I'd link this to Github, yes.
Usability-wise it would be cool if this could be direct links as well. Something like:

```
Download now for [Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe), [Mac](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_mac.dmg), [Debian/Ubuntu](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb) or [other platforms](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0)
```

Which would render as

Download now for [Windows](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_win.exe), [Mac](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_mac.dmg), [Debian/Ubuntu](https://github.com/jamulussoftware/jamulus/releases/download/r3_7_0/jamulus_3.7.0_ubuntu_amd64.deb) or [other platforms](https://github.com/jamulussoftware/jamulus/releases/tag/r3_7_0)

Reply:
Yes, that would be better I think at least until we have a "download" page which can provide some more context, changelog, whatever.

Reply:
I have opened https://github.com/jamulussoftware/jamuluswebsite/issues/359 to track the update of download links.